### ISSUE TITLE
infomap via `conda-forge`?

### ISSUE THREAD

**Issue by jGaboardi** [ulfaslak/infostop#24]
I am wondering if there is consideration for making `infostop` available via [`conda-forge`](https://conda-forge.org/#page-top)? Having this package available there would make it much more accessible within the scientific Python community and increase its visibility.

cc @jrmcgaha-dev

**Comment by ulfaslak** [comment]
Good point. I can't dedicate much time to side projects these days (two small kids and full time job).

As I understand it, [putting it on conda-forge is simple enough](https://chat.openai.com/share/2c6614b2-d60e-4fac-ae09-3c564fdb505e). Make a PR to `conda-forge/staged-recipes`, with a meta.yaml recipe file for the package, and then hope 🤞 the package passes review. The time consuming part I don't have time for it going back and forth with the conda-forge review team to get it accepted. This package lacks proper unit- and integration tests, and probably has other problems that I wasn't mature enough as a developer to resolve back when I wrote it.

You are however VERY welcome to start the process of putting it on conda-forge (make a PR for it, I promise I will review it), and if there is anything I need to do to facilitate that process (which takes less than an hour) I will gladly do it!
